From f8d033b37b5d449a13de78f4288c915de7881fdc Mon Sep 17 00:00:00 2001
From: Martin Liu <liumartin@google.com>
Date: Tue, 12 Sep 2023 16:02:17 +0800
Subject: [PATCH 2/3] sm8750-common: init: decrease swappiness to 60

This tweak also aligns the policy with P23.

Test: boots with correct settings
Bug: 299975853
Change-Id: I0ed7088ddf2ff2fa0feca2197e5cc03c7f5c6778
Signed-off-by: Martin Liu <liumartin@google.com>
---
 init/init.kernel.post_boot-memory.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/init.kernel.post_boot-memory.sh b/init/init.kernel.post_boot-memory.sh
index 1451d71..042dc4c 100755
--- a/init/init.kernel.post_boot-memory.sh
+++ b/init/init.kernel.post_boot-memory.sh
@@ -183,7 +183,7 @@ function configure_memory_parameters() {
 	# Call configure_min_free_kbytes after
 	configure_min_free_kbytes
 
-	echo 100 > /proc/sys/vm/swappiness
+	echo 60 > /proc/sys/vm/swappiness
 
 	#Set per-app max kgsl reclaim limit and per shrinker call limit
 	if [ -f /sys/class/kgsl/kgsl/page_reclaim_per_call ]; then
-- 
2.55.0

